lib/matplotlib/tests/test_triangulation.py

```diff
@@ -19,9 +19,9 @@ def test_delaunay():
     x, y = np.meshgrid(np.linspace(0.0, 1.0, nx), np.linspace(0.0, 1.0, ny))
     x = x.ravel()
     y = y.ravel()
-    npoints = nx*ny
-    ntriangles = 2 * (nx-1) * (ny-1)
-    nedges = 3*nx*ny - 2*nx - 2*ny + 1
+    npoints = nx * ny
```

Comment:
Please don't make unrelated changes to the file.

Reply:
Yes, Sorry for that.